homevork/05-redux-basic/001-CombineRedecer/index.ts

```diff
@@ -0,0 +1,86 @@
+type Action = {
+  type: string;
+  payload?: any;
+};
+
+type Store<T> = {
+  getState: () => T;
+  dispatch: (action: Action) => void;
+};
+
+function counterOne(state: number = 0, action: Action) {
+  switch (action.type) {
+    case "INCREMENT":
+      return state + 1;
+    case "DECREMENT":
+      return state - 1;
+    default:
+      return state;
+  }
+}
+function counterTwo(state: number = 0, action: Action) {
+  switch (action.type) {
+    case "INCREMENT_TWICE":
+      return state + +2;
```

Comment:
`state + +2`? what is reason for that?

Reply:
Action called INCREMENT_TWICE.That action should increment two times  that means add 2.